android/assets/jsons/translations/French.properties

```diff
@@ -713,23 +692,20 @@ mounted units = unités montées
 naval units = unités navales
 ranged units = unités à distance
 # For the All "newly-trained [relevant] units in this city receive the [] promotion" translation. Relevant as in 'units that can receive'
-relevant = concerné
-non-air = non-air
+relevant = concernées
+non-air = non-aériennes
 # For '[stats] from [Water] tiles in this city'
 Water = Eau
 # For [stats] from [Water resource] tiles in this city
 Water resource = Ressource aquatique
 River = Rivière
-Fresh water = Eau douche
-non-fresh water = Eau non-douce
- # Requires translation!
-Coastal = 
- # Requires translation!
-Bonus resource = 
- # Requires translation!
-Strategic resource = 
- # Requires translation!
-Luxury resource = 
+Fresh water = Eau douce
+non-fresh water = Eau salée
+# That's an adjective. What is it applied to? I need to find out whether it's masculine.
+Coastal = Côtier(e)
```

Comment:
I would really like to know the gender. Is that a Coastal tile? Resource? Or should I keep it generic?

Reply:
Based on the context (but I don't know this for sure) I would say that it would appear in contexts like: ```+2 food from coastal tiles```, where coastal means land tiles adjacent to the coast.